Mailer/app/models/Repositories/LogsRepository.php

```diff
@@ -28,6 +28,11 @@ class LogsRepository extends Repository
         'suppress-unsubscribe' => 'hard_bounced_at',
     ];
 
+    public function all()
+    {
+        return $this->getTable()->select('*')->order('created_at DESC');
```

Comment:
- Don't enforce order which is not universal for all. Someone might want to order things different way.
- Why the `select(*)`?